lib/ZcapClient.js

```diff
@@ -132,8 +132,8 @@ export class ZcapClient {
       };
     }
 
-    if(allowedActions.length > 0) {
-      delegatedCapability.allowedActions = allowedActions;
+    if(allowedAction.length > 0) {
+      delegatedCapability.allowedAction = allowedAction;
```

Comment:
Actually, I recommend we only change this one line. The data model uses `allowedAction` as that is idiomatic there, but using plural form in the API is also idiomatic. So I suggest this PR just adjust the one line here to use singular on `delegatedCapability`, leaving the docs and API the same.